src/app/routes/utils/fetchPageData/index.js

```diff
@@ -67,12 +67,11 @@ const handleError = (e) => {
   };
 };
 
-const fetchData = (pathname) => {
-  const url = getUrl(pathname);
-
+export const fetchData = (url) => {
```

Comment:
We wanted to reuse all of the data fetching logic, but pass in an endpoint that was unrelated to the logic handled by `getUrl`. We should rearrange the tests for fetchPageData to test the fetchData function if we stick with this.

Reply:
Taking another look at this, if we want to handle errors differently, which we probably do, then we don't want to reuse this function. Have added the data fetching code as a function into the withRadioSchedules file which means we can handle errors with a different constant, and ignore all of the 'error propagation' that we don't need for this call.